main/tests/server/src/com/google/refine/grel/controls/ForRangeTests.java

```diff
@@ -27,15 +27,55 @@
 
 package com.google.refine.grel.controls;
 
+import com.google.refine.RefineTest;
+import com.google.refine.expr.*;
+import com.google.refine.model.Project;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 
 import com.google.refine.util.TestUtils;
 
-public class ForRangeTests {
+import java.util.Properties;
+
+import static org.testng.Assert.assertTrue;
+import static org.testng.Assert.fail;
+
+public class ForRangeTests extends RefineTest {
 
     @Test
     public void serializeForRange() {
         String json = "{\"description\":\"Iterates over the variable v starting at \\\"from\\\", incrementing by \\\"step\\\" each time while less than \\\"to\\\". At each iteration, evaluates expression e, and pushes the result onto the result array.\",\"params\":\"number from, number to, number step, variable v, expression e\",\"returns\":\"array\"}";
         TestUtils.isSerializedTo(new ForRange(), json);
     }
+
+    @Test
+    public void testForRangeArray() throws ParsingException {
+        String test[] = { "forRange(10.5,8,-0.5,v,v).join(',')", "10.5,10.0,9.5,9.0,8.5" };
```

Comment:
Please add a test or two for things like `from == to`, `step > from - to`, and other edge cases. The code is pretty simple, so but it's a good habit to get into to test the corners.